ASU-OKED-WEB-DEV/Neuroscience-scholars
# Drupal update git init

## Description
An updated composer.lock file that contains most recent Drupal updates.
Added to GIt (Contains first commit) and added .lando.yml file.

## Deployment notes
1. Run, `git clone https://github.com/ASU-OKED-WEB-DEV/Neuroscience-scholars.git`.
2. Run, `cd Neuroscience-scholars`.
3. Run, `lando start`. Start this local instance with prod DB, files directory and settings file.
4. Run, `composer install`.
5. Run, `git fetch --all`.
6. Run, `git checkout drupal-update`.
7. Run, `composer install`.
8. Run, `drush updb`.
9. Run, `drush cr`.

## Instructions to test
1. Run, `drush status`, ensure Drupal core is at 8.6.2.
2. Check website to ensure all pages are working and rendering correctly.
